serdes/jsonschema-serde/src/main/java/io/apicurio/registry/serde/jsonschema/JsonSchemaParser.java

```diff
@@ -64,8 +60,22 @@ public ParsedSchema<JsonSchema> getSchemaFromData(Record<T> data, boolean derefe
         return null;
     }
 
+    @Override
+    public ParsedSchema<JsonSchema> getSchemaFromLocation(String location) {
```

Comment:
I did not know (from the docs or the property) what is mean by location. It is loading the file from the classpath, but user could be confused and try to load file from the filesystem or URL.
I would suggest updating the config property documentation, and/or accepting normal file path and/or validating it.

Reply:
Ok, I will make it clear in the docs that it's the classpath, since that is the intention here and what will be done in Quarkus.